tests/integration/test_vulnerability_detector/test_scan_results/data/test_cases/cases_scan_provider_vulnerabilities.yaml

```diff
@@ -59,3 +59,15 @@
     nvd_feed: 'custom_nvd_alternative_feed.json'
     json_feed_tag: CUSTOM_ARCH_JSON_FEED
     nvd_feed_tag: CUSTOM_NVD_JSON_FEED
+
+- name: 'Ubuntu22'
+  description: 'Scan Ubuntu22 vulnerabilities using provider and NVD feed'
+  configuration_parameters: null
+  metadata:
+    provider_name: 'canonical'
+    system: 'JAMMY'
+    json_feed: null
+    oval_feed: 'custom_ubuntu22_oval_feed.xml'
+    nvd_feed: 'custom_nvd_alternative_feed.json'
+    oval_feed_tag: CUSTOM_JAMMY_OVAL_FEED
+    nvd_feed_tag: CUSTOM_NVD_JSON_FEED
```

Comment:
This section should be removed because for canonical the latest version is chosen to test, so you should change the line 32(where the system is `focal `to `jammy`)